zadaci/sa-kolokvijuma/2020/PSI/T5/G1/index.rst

```diff
@@ -0,0 +1,17 @@
+.. index::
+   Datoteke ; Tekstualne
+   Dinamičke strukture ; Lista
+
+
+Zadatak "Najuspešnija knjiga"
+=================
```

Comment:
Line 7 has to be as long as line 6 by RestructuredText specs. Please correct this.